handlers/expenses.ts

```diff
@@ -0,0 +1,63 @@
+import { Expense, Transaction } from "../helpers/calculateExpenses";
+interface IExpenses {
+    getExpenses(groupId: string, callback: (expenses: Expense[]) => void): void;
+    settleExpenses(groupId: string, transaction: Transaction): void;
+    getGroupMembers(groupId: string, callback: (users: string[]) => void): void
+}
+
+class ExpensesHandle implements IExpenses{
+
+    constructor(private gun: Gun) {
```

Comment:
Either set a this.gun and use that pattern like in the other handlers, or remove the need for passing gun to the constructor, in which case you should't need any constructor as typescript has a default